skyportal/plot.py

```diff
@@ -1343,6 +1344,34 @@ def photometry_plot(obj_id, user, width=600, device="browser"):
     return bokeh_embed.json_item(tabs)
 
 
+def smoothing_function(values, window_size):
+    """
+    Smooth the input "values" using a rolling average
+    where "window_size" is the number of points to use
+    for averaging.
+    This should be the same logic as static/js/plotjs/smooth_spectra.js
+    """
+
+    if values is None or not hasattr(values, '__len__') or len(values) == 0:
```

Comment:
What does this code do?

Reply:
Not sure what you mean? 
On line 1355 I just check that I didn't get `None` or an empty array or something that isn't an array or list.

Reply:
OK